spec.html

```diff
@@ -23,16 +23,16 @@ <h1>Syntax</h1>
     ImportDeclaration :
       `import` ImportClause FromClause `;`
       `import` ModuleSpecifier `;`
-      <ins>`import` ImportClause FromClause WithClause `;`</ins>
-      <ins>`import` ModuleSpecifier WithClause `;`</ins>
-      <ins>`export` ExportFromClause FromClause WithClause `;`</ins>
+      <ins>`import` ImportClause FromClause IfClause `;`</ins>
+      <ins>`import` ModuleSpecifier IfClause `;`</ins>
+      <ins>`export` ExportFromClause FromClause IfClause `;`</ins>
 
-    WithClause :
-      <ins>`with` `{` WithEntries `}` `;`</ins>
+    IfClause :
+      <ins>`if` `{` IfEntries `}` `;`</ins>
 
-    WithEntries :
+    IfEntries :
```

Comment:
Editorial: Let's rename this production to something like `AttributeEntries`, so it makes more sense to reuse as-is if we add `with`-attributes later.